doc/src/modules/stats.rst

```diff
@@ -117,8 +117,6 @@ Interface
 .. autofunction:: std
 .. autofunction:: sample
 .. autofunction:: sample_iter
-.. autofunction:: sympy.stats.rv.sample_iter_lambdify
-.. autofunction:: sympy.stats.rv.sample_iter_subs
```

Comment:
Probably the failure of https://travis-ci.org/github/sympy/sympy/jobs/677325425 is due to this change. Have you tried building the docs locally.
ping @sylee957 Can you please help regarding the SPHINX failure? Thanks.

Reply:
I think that the HTML build succeeded but have only found the abnormality in doctr. So it can be temporary.
But I think that author needs to resolve the merge conflicts to see if it works for an another trial.